src/main/java/uk/gov/companieshouse/api/accounts/transaction/TransactionManager.java

```diff
@@ -5,4 +5,6 @@
 public interface TransactionManager {
```

Comment:
I was under the impression all the classes in the transaction package were copied form the Transaction API and were to be deleted at some stage. If we start changing them don't we run the risk of adding functionality that prevents us from doing this?

Reply:
Which parts are you referencing has been changed?